addons/mail/wizard/mail_template_preview_views.xml

```diff
@@ -24,7 +24,10 @@
                         <div class="row">
                             <span class="col-md-5 col-lg-4 col-sm-12 ps-0">Force a language: </span>
                             <div class="col-md-7 col-lg-6 col-sm-12 ps-0">
-                                <field name="lang" placeholder="Select a language"/>
+                                <div class="d-flex">
+                                    <field name="lang" placeholder="Select a language" class="flex-grow-1"/>
+                                    <div class="d-inline px-1"/>
```

Comment:
mhe, it's a bit tricky and might be removed if someone read the file

can't we put the margin directly on the field itself ?

Reply:
Thanks for the review. I totally agree with you and actually to fix the problem, adding w-100 on the 2 fields solves the problem of the selection not being long enough to see all the text. 

There was an extra bug discovered during the testing that is that the 2 selections are not perfectly of the same length. As explained in the commit and the task, this is because one of the field widget implementation add a div of 5 pixel just after the selection (empty div with margin). I have tried already a few hours to solve it correctly (trying margin padding, … on the field but it doesn’t work because we have to put w-100 and the margin on the field tag, we cannot go below with just html). But I think as I have mentioned that the only way to solve it correctly is in the widget itself. Even the solution with margin (that I couldn’t make work) will cause a bug if the field widget is corrected to remove that div of 5 pixel.

Actually, I found 2 solutions that works, this one and adding a css file that target that specific extra div and set its margin to 0 (with important), only for this form. But it is also a hack and it is less localized (it is in another css file).

Maybe you have an idea to solve it more properly ? Otherwise, either we may skip this second bug as it is very minor or solve it in the field widget implementation. 

What do you think? Maybe adding a comment ?

Reply:
> Actually, I found 2 solutions that works, this one and adding a css file that target that specific extra div and set its margin to 0 (with important), only for this form. But it is also a hack and it is less localized (it is in another css file).

Actually I prefer this solution :)